### ISSUE TITLE
Resulting folder from NUS download doesn't contain "content" or "audio" folders?

### ISSUE THREAD

**Issue by Truanifan** [MrMysterio/Uwizard#7]
I’m not entirely sure what’s going wrong or how best to explain it.
I think it’s downloading from NUS just fine since the resulting folder in 6+GB, but for some reason the output folder doesn’t seem to have the same files/folders as in the video I watched.

My goal is to extract select sound files, but I don’t see an “audio” folder, let alone a “content” folder, in the nus_content folder.

Example:
![uwiz2](https://cloud.githubusercontent.com/assets/10821488/12023936/d59b77fc-ad6c-11e5-9254-85969383b909.jpg)

When I open the nus_content folder all I see are 2 folders with a series of numbers. One of which is totally empty and the other has a series of numbered files and a “tmd” file.
![uwiz1](https://cloud.githubusercontent.com/assets/10821488/12023934/ce2ab56e-ad6c-11e5-87bb-ded0729e5ec6.jpg)

My system is running Windows 7 and is 64-bit. I’m not very computer savvy so please explain anything I can do in layman’s terms.

I really appreciate any help with this. I bought the game and just wanted to get some sounds and music off it. Thanks in advance.

**Comment by colt05-usr** [comment]
Sorry for late reply
There is a "Decrypt Contents" check box next to the version box. The numbered files are encrypted contents. You will need a title key to decrypt the contents though.
